src/composables/PluginManager.js

```diff
@@ -1,6 +1,6 @@
 import { readTextFile } from 'src/composables/Files';
 import plugins from 'src/plugins';
-import { FileInput, FileInformation } from 'leto-modelizer-plugin-core';
+import { FileInput, FileInformation, ComponentDrawOption } from 'leto-modelizer-plugin-core';
```

Comment:
is after 3 elements I think.

Reply:
Because linter is configured to have 3 maximum properties in curly brackets

```js
{ 1, 2, 3 }      // right

{ 1, 2, 3, 4 }   // wrong

{                // right
  1,
  2,
  3,
  4,
}
```

Reply:
Done